cpp/meson.build

```diff
@@ -56,7 +56,9 @@ if git_description == ''
     git_description = run_command('git', 'describe', '--tags', check: false).stdout().strip()
 endif
 
+needs_acero = false
```

Comment:
Can we add this when we add `acero/` directory?

Reply:
No problem. I added it here since `needs_compute` will eventually depend on it, but we can add that logic later